go.mod

```diff
@@ -5,7 +5,7 @@ go 1.13
 require (
 	github.com/aquasecurity/fanal v0.0.0-20200427221647-c3528846e21c
 	github.com/aquasecurity/go-dep-parser v0.0.0-20190819075924-ea223f0ef24b
-	github.com/aquasecurity/trivy-db v0.0.0-20200430091154-7c0a6e1ad398
+	github.com/aquasecurity/trivy-db v0.0.0-20200430232549-8078bee954c5
```

Comment:
Note: update this to master when https://github.com/aquasecurity/trivy-db/pull/43 is merged.